Form/Type/ResetPasswordType.php

```diff
@@ -44,6 +65,21 @@ public function __construct(Security $security, string $userClass)
      */
     public function buildForm(FormBuilderInterface $builder, array $options): void
     {
+        $notPasswordCompromised = null;
+        $passwordStrength = new PasswordStrength([
```

Comment:
Zou je hier niet even moeten afvangen of minStrength en minLength een waarde hebben ?

Reply:
Er zit een default waarde in minstrength en minlength, dus leeg zouden ze inprincipe nooit zijn